backend/src/main/java/com/github/jekattack/cornergame/game/visits/VisitController.java

```diff
@@ -17,8 +17,10 @@ public class VisitController {
     private final VisitService visitService;
 
     @PostMapping("/add")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
you specify the status created later in the try segment. So no need for the annotation